bdai_ros2_wrappers/bdai_ros2_wrappers/tf_listener_wrapper.py

```diff
@@ -4,25 +4,24 @@
 
 import rclpy
 from geometry_msgs.msg import TransformStamped
-from rclpy import Context
+from rclpy.node import Node
 from rclpy.time import Duration, Time
 from tf2_ros import ExtrapolationException, TransformException
 from tf2_ros.buffer import Buffer
 from tf2_ros.transform_listener import TransformListener
 
-from bdai_ros2_wrappers.node import NodeWrapper
+import bdai_ros2_wrappers.process as process
 
 
 class TFListenerWrapper(object):
     def __init__(
-        self,
-        node_name: str,
-        wait_for_transform: Optional[Tuple[str, str]] = None,
-        cache_time_s: Optional[int] = None,
-        context: Optional[Context] = None,
+        self, node: Node, wait_for_transform: Optional[Tuple[str, str]] = None, cache_time_s: Optional[int] = None
     ) -> None:
-        # private because we want to make sure no one but us spins this node!
-        self._node = NodeWrapper(node_name=node_name, context=context, spin_thread=True)
+        node = node or process.node()
```

Comment:
Evaluation to a Boolean?

Reply:
This one's a handy gimmick. Values are coerced to boolean by the operator, but the expression yields the original value. So `None or 'test'` yields `'test'`.